plugin/src/patcher.ml4

```diff
@@ -133,17 +133,24 @@ let patch env n try_invert a search sigma (define, last_def) =
 
 (* Defines a patch as a new hypothesis. *)
 let patch_def_hypothesis =  
-    ((fun n sigma patch last_def ->
-      Proofview.tclBIND last_def
-        (fun _ -> letin_pat_tac false None (Names.Name n)
-                    (sigma, EConstr.of_constr patch) Locusops.nowhere)),
-     Tacticals.New.tclIDTAC)
-
+  ((fun n sigma patch last_def ->
+    Proofview.tclBIND last_def (fun _ ->
+        letin_pat_tac false None (Names.Name n)
+          (sigma, EConstr.of_constr patch) Locusops.nowhere)),
+   Tacticals.New.tclIDTAC)
+  
+(* Suggest something to do with the generated patch. *)
+let patch_suggest =
+  ((fun _ _ patch last_def ->
+    let s = Printer.pr_constr_env (Global.env ()) Evd.empty patch in
+    Feedback.msg_notice (str "apply " ++ s); last_def),
```

Comment:
Sorry if I told you to use the wrong one (I very well may have), but you want `msg_info` rather than `msg_notice` for consistent behavior across IDEs